webapp/lib/controller.dart

```diff
@@ -128,25 +143,50 @@ void command(UIAction action, Data actionData) {
       ProjectConfigurationData projectConfigurationData = actionData;
       saveProjectConfiguration(projectConfigurationData.config);
       break;
+    case UIAction.saveProjectConfiguration:
+      savePackageConfiguration();
+      break;
+  }
+}
+
+enum NavAction {
+  ALLPROJECTS,
+  DASHBOARD
+}
+
+void _toggleNavActions(NavAction navAction, bool show) {
+  switch (navAction) {
+    case NavAction.ALLPROJECTS:
+      view.navView.allProjectsLink.classes.toggle('nav-links__link--show', show);
+      break;
+    case NavAction.DASHBOARD:
+      view.navView.dashboardLink.classes.toggle('nav-links__link--show', show);
+      break;
   }
 }
 
 void loadAuthView() {
+  _toggleNavActions(NavAction.ALLPROJECTS, false);
+  _toggleNavActions(NavAction.DASHBOARD, false);
```

Comment:
the repetition of these two lines is going to be quite error prone, and hard to read.

I suggest that instead the `navView` keeps a `Map<NavAction, link>` and has a method (as per above) that takes a `NavAction` and then toggles the classes appropriately. In the controller that map should be instantiated on view init, and then that method gets called (here and in the other methods below) when the nav elements needs to change

Reply:
Implemented this